### ISSUE TITLE
Merge context and profile settings into "About" settings

### ISSUE THREAD

**Issue by SimoneZaza** [alkem-io/client-web#6878]
## Description

As a user, it is currently not very clear to me what this context page is, where it's content is visible, and where I can edit the things I see in the about dialog. 

## Acceptance criteria
For both Spaces and Subspaces,
- [ ] Move Context fields (which are changed to About fields on the server) to be part of the profile page, as per design
- [ ] Remove Context page from settings
- [ ] Rename "Profile" tab in Settings to be "About"
- [ ] New fields are:
- What (= profile description)
- Why (= why, former impact)
- Who (= who, former who)
- Note that the 3rd context field (background) disappeared. The content will be merged with the Why in a migration (separate story)

## Additional Context
Design: https://www.figma.com/design/9PtBSGJyv3y1dRpGJStj8e/Join-Apply-Button-More-Prominent?node-id=55-2&t=jvgKIkALJhSof64H-1
Context: https://alkem.io/building-alkemio-org/challenges/improvingtheusere-6400/collaboration/profilevscontext

## Areas that will be affected 
 
_To be added during the refinement_

**Comment by DonnaDuiker** [comment]
I agree with moving the context to the profile page in the settings, would be way more clear!

However, the word "profile" I associate more with a person/user, so I personally prefer the button how it is now (About this space) instead of profile.

And some helper text next to what, why, who and when/where would be good if this was not the plan already, it feels a bit blunt like this.

**Comment by DonnaDuiker** [comment]
Yes

helper texts:

Vision --> What

- What is this community about?

Impact --> Why

- What will this Space change? How will people and organizations be affected?

Who --> Who

- Who should be involved?

Background --> When/Where

- Give a brief overview of the need for this Space and how it came about.

These are the current ones and I think they work!

**Comment by larssondenise** [comment]
The When/where title does not always fit, but the description does (the need for this space).

 How much do we care about old communities? @SimoneZaza

**Comment by Comoque1** [comment]
Tested OK on all journey levels with respect to this reference:
<img width="847" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFlvQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9139d165562ff9eeb9eddf6b94e4831accd792e5/image.png" alt="image.png" />

<img width="669" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNGNvQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--67e927c02a64ca5683286c2fb584c9b015deba9a/image.png" alt="image.png" />

<img width="751" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNGdvQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7ec390aa51fb137c53aa6500a881348d4b0c3e2b/image.png" alt="image.png" />
